clj-refactor.el

```diff
@@ -149,6 +149,12 @@ as can be."
   :group 'cljr
   :type 'boolean)
 
+(defcustom cljr-eagerly-cache-macro-occurrences-on-startup t
```

Comment:
can take a long time and eats quite a bit of CPU at REPL startup. some users might not want this for some reason...